lib/pbench/test/unit/server/query_apis/test_iteration_samples.py

```diff
@@ -0,0 +1,320 @@
+import pytest
+from http import HTTPStatus
+from pbench.server.api.resources.query_apis.iteration_samples import (
+    IterationSampleNamespaces,
+)
+from pbench.test.unit.server.headertypes import HeaderTypes
+from pbench.test.unit.server.query_apis.commons import Commons
+
+
+class TestIterationSamplesRows(Commons):
+    """
+    Unit testing for IterationSamplesRows class.
```

Comment:
Class name and comment are wrong; we're testing the "namespace" API, not the "rows" API!